### DISCUSSION TITLE
Recover my account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#112409] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a Github account with username alkapriya-infobahn. I was not working from last 7 months due to medical issue. some how in my Authenticator app 2f authenticator has been logout and also, I don't have recovery code.
Please recover my account as I am not able to login to my Github account and because of that my work has been blocked. 
Request you to kindly help me to recover my account.

**Comment by AidenJamesYt** [comment]
Been there, done that. Here is what i did:

Visit the GitHub Account Recovery Page:

Go to the GitHub account recovery page (its https://github.com/recover)
Enter the username or email address associated with your GitHub account, then follow the account recovery instructions

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @alkapriya-ib! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, our [GitHub Account Recovery Policy](https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy) should have the answer that you're looking for. 

I've gone ahead and moved this to the correct category for you. Good luck!
